search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt

```diff
@@ -58,7 +67,7 @@ class SearchAudioDetailViewModel @Inject constructor(private val analyticsTracke
 
         articObjectObservable
                 .map { it.location.orEmpty() }
-                .map { it.isEmpty() }
+                .map { !it.isEmpty() }
```

Comment:
We could use `it.isNotEmpty()` too, right?

Reply:
true. updated